auction-server/src/config.rs

```diff
@@ -62,6 +62,11 @@ pub struct RunOptions {
     #[arg(env = "SUBWALLET_PRIVATE_KEY")]
     pub subwallet_private_key: String,
```

Comment:
maybe prefix with `evm`?

Reply:
we can do this in a separate refactor as Dani suggested. I didn't want to break backward compatibility here.